CHANGELOG.md

```diff
@@ -6,6 +6,168 @@
 - **Docker**: Our Ubuntu based images have been upgraded to Ubuntu [20.04 LTS](https://releases.ubuntu.com/20.04/).
 - **Dashboard**: A global minimum dashboard refresh interval is now enforced and defaults to 5 seconds.
 
+# 7.0.0-beta1 (2020-04-28)
+
+## Breaking changes
+
+- **Removed PhantomJS**: PhantomJS was deprecated in [Grafana v6.4](https://grafana.com/docs/grafana/latest/guides/whats-new-in-v6-4/#phantomjs-deprecation) and starting from Grafana v7.0.0, all PhantomJS support has been removed. This means that Grafana no longer ships with a built-in image renderer, and we advise you to install the [Grafana Image Renderer plugin](https://grafana.com/grafana/plugins/grafana-image-renderer).
+- **Docker**: Our Ubuntu based images have been upgraded to Ubuntu [20.04 LTS](https://releases.ubuntu.com/20.04/).
+- **Dashboard**: A global minimum dashboard refresh interval is now enforced and defaults to 5 seconds.
+### @grafana/ui forms migration notice
```

Comment:
@tskarhed what do you think if we add a link from here to grafana/ui changelog?

Reply:
Yeah, that would work too

Reply:
```suggestion
- **@grafana/ui**: Forms migration notice, see [@grafana/ui changelog](https://github.com/grafana/grafana/blob/master/packages/grafana-ui/CHANGELOG.md)

Reply:
@tskarhed added suggestions